integration_test/chart_values/backend.yaml

```diff
@@ -0,0 +1,2002 @@
+## Values configuration for Mojaloop Backend Dependencies
+# helm repo add bitnami https://charts.bitnami.com/bitnami
+# helm repo add cloudhut https://raw.githubusercontent.com/cloudhut/charts/master/archives
+# helm install backend ./dependencies/backend
+# helm upgrade backend ./dependencies/backend --install
+
+global:
+
+kafka:
+  enabled: false
+
+################################################################
+## MySQL Backend Dependency
+mysql:
+  enabled: true
```

Comment:
Are you using this mysql service? If not you can disable this.

Reply:
I'll do this in another PR.